rfd/0182-multi-port-tcp-app-access.md

```diff
@@ -544,8 +544,7 @@ Changes in `api/proto/teleport/legacy/client/proto/authservice.proto`:
 +  // Used merely for audit events and mirrors the URI from the app spec.
    string URI = 8 [(gogoproto.jsontag) = "uri,omitempty"];
 +  // TargetPort signifies that the cert grants access to a specific port in a multi-port TCP app, as
-+  // long as the port is defined in the app spec. When specified, it must be between 1 and 65535 and
-+  // the URI is expected to use this port as well.
```

Comment:
It isn't true that the URI is expected to include TargetPort. This is a section that I forgot to update in the RFD PR when [I was making this change](https://github.com/gravitational/teleport/pull/46169/commits/d172c580f98192f181d9b5c7abdb039902ca3ff2#diff-3aef5deb27cb4f67a335353505a67cf64f157bee946f3aca8817ef5179323444L408-L412).

Also wow, whatever GitHub uses for syntax highlighting breaks terribly when the things that being changed uses diff syntax. :~)